### PR TITLE
Update tech-spec.md

### PR DESCRIPTION
**Pull request by cmsdroff**
Small mod to text to suggest to `tag` the official releases in GitHub for stability.  Commented in #4 as it doesn't appear that we have applied in the current outputs.

### CODE REVIEW

**Comment by cmsdroff** [comment]
updated the tech spec to cover a few points in #10 #9 #4

File: docs/tech-spec.md [hunk 1]
```diff
@@ -156,35 +156,44 @@ It has been agreed that the scope will NOT cover
...
+2. Code in the UN/CEFACT GitHub (https://github.com/uncefact/vocab) to produce the JSON-LD vocabulary outputs
+3. Delivery of an official and maintained release of the JSON-LD vocabulary published on the UNECE website (https://service.unece.org/trade/uncefact/vocabulary/uncefact.jsonld)
 
 
 ## Maintenance
-Release Cycles and practicalities.
 
+UN/CEFACT release their reference data models twice a year (usually around July and December) the json-ld vocab would be produced as a follow on output from the release of Buy Ship Pay (BSP) reference data model.
+
+1. BSP RDM Release
```
Comment (nissimsan) [inline docs/tech-spec.md:168]: @cmsdroff, some of this needs discussion. I worry this is promising a bit too much.

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: docs/tech-spec.md (hunk 1 above)
Comment (cmsdroff) [reply docs/tech-spec.md:168]: Hi @nissimsan as I understand CEFACT deliverables are twice a year so to be official outputs they would need to be part of this release schedule.

We have an open ticket #87 which I think probably need to be resolved before we include the text above if your not comfortable with it at this stage.  So happy to remove it for time being?

Comment (cmsdroff) [reply docs/tech-spec.md:168]: @nissimsan changes made for review on next call

**Review by nissimsan — APPROVED** [review]

**Pull request merged**